CODEOWNERS

```
@@ -11,12 +11,12 @@ VERSION     @digital-asset/empty
 /3rdparty/          @digital-asset/empty
 /bazel_tools/       @digital-asset/empty
 /build-scripts/     @digital-asset/empty
-/ci/                @digital-asset/empty
-/dev-env/           @digital-asset/empty
+/ci/                @garyverhaegen-da
```

I think you're supposed to keep `@digital-asset/empty` so those who want to be notified of everything can be members of the empty group. (!!)

```suggestion
/ci/                @digital-asset/empty @garyverhaegen-da
```

Same goes for the other changes.

---

That was not the intention, just meant as a placeholder, but we can set something like that up. I don't think we should use CODEOWNERS for notifications though, since it conflates two separate concerns.

---

Ok, approving then.

---

(But note that there are lines with `@digital-asset/empty @gerolf-da` for instance.)

---

Yep, that's fair. I expect to go through this in a couple of weeks or so and tidy it up after people have had a chance to add themselves as they prefer (and I would then remove the team from the org as well).

---

AFAIK, `CODEOWNERS` are the only way in GitHub to tailor what notifications you get. Thus, I'd be very much in favour if nobody removed the `@digitial-asset/empty` entries but just adds themselves to the list. We should probably rename the group into `@digital-asset/default` or something similar.

---

perhaps `@digital-asset/all-notifications`

---

If you want all notifications you can just subscribe to the repo without code owners. That's what I've done and it works well. As a result, I think eliminating empty is probably fine.